client/lib/widgets/top_nav.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:client/constants/resources.dart';
+import 'package:client/themes/theme_constants.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:responsive_builder/responsive_builder.dart';
+
+final GlobalKey<ScaffoldState> scaffoldKey = GlobalKey();
+AppBar topNavigationBar(BuildContext context, GlobalKey<ScaffoldState> key) =>
```

Comment:
Why is-this a method? It should be a class!

Reply:
u mean appbar inside class.

Reply:
Yes

Reply:
I'm getting this error 
![image](https://user-images.githubusercontent.com/96618001/176998932-c5d739c1-b0c0-4dff-9523-697fe3ae7a76.png)